tests/toplevel_unit.c

```diff
@@ -4572,10 +4572,26 @@ DEFINE_TEST(convert_to_bitset) {
     roaring_bitmap_free(r1);
 }
 
+// simple execution test
+DEFINE_TEST(simple_roaring_bitmap_or_many) {
+    roaring_bitmap_t ** roaring_bitmaps;
+    roaring_bitmaps = (roaring_bitmap_t**) malloc(2 * sizeof(roaring_bitmap_t*));
+    roaring_bitmaps[0] = roaring_bitmap_create();
+    roaring_bitmaps[1] = roaring_bitmap_create();
```

Comment:
nit: I would use an array here, just `roaring_bitmap_t *roaring_bitmaps[2] = {roaring_bitmap_create(), roaring_bitmap_create()};`

Reply:
I might do so.

This a copy-paste from an informal bug report.